tox.ini

```diff
@@ -45,6 +45,20 @@ python =
     3.11: py311
     3.12-dev: py312
 
+[testenv:mypy]
+setenv =
+  PYTHONPATH = {env:PATH}/testapp
+deps =
+  {[testenv]deps}
+  django-stubs[compatible-mypy]
+  types-stripe
+  types-requests
+  types-braintree
+  types-xmltodict
+  types-dj-database-url
+extras = {[testenv]extras}
+commands = mypy .
```

Comment:
These are here only temporarily; I intend to move all dependencies into `pyproject.toml`.